inlayHints.py

```diff
@@ -0,0 +1,116 @@
+from .protocol import InlayHint, InlayHintRequestParams, InlayHintResponse
+from html import escape as html_escape
+from LSP.plugin import Request
+from LSP.plugin import Session
+from LSP.plugin.core.protocol import Point
+from LSP.plugin.core.registry import windows
+from LSP.plugin.core.types import debounced
+from LSP.plugin.core.types import FEATURES_TIMEOUT
+from LSP.plugin.core.typing import List, Optional, Tuple
+from LSP.plugin.core.views import point_to_offset
+from LSP.plugin.core.views import region_to_range
+from LSP.plugin.core.views import text_document_identifier
+import sublime
+import sublime_plugin
+
+
+def inlay_hint_to_phantom(view: sublime.View, hint: InlayHint) -> sublime.Phantom:
+    html = """
+    <body id="lsp-typescript-inlay-hints">
+        <style>
+            body {{
+                padding: 0;
+                margin: 0px;
+                border: 0px;
+                font-size: 0.9em;
+            }}
+
+            .lsp-typescript-inlay-hints {{
+                color: color(var(--foreground) alpha(0.6));
+                background-color: color(var(--foreground) alpha(0.08));
+                border-radius: 4px;
+                padding: 2px;
+            }}
+        </style>
+        <div class="lsp-typescript-inlay-hints">
```

Comment:
But then this class doesn't have to be this specific if the idea is to target those using `#id #class` selector. This could just be `inlay-hint`, for example.

Reply:
Looks good.